.github/workflows/release-apps.yml

```diff
@@ -0,0 +1,75 @@
+name: Release Apps
+
+on:
+  release:
+    types: [ published ]
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Publish
+    if: github.event.pull_request.head.repo.full_name == github.repository
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        app: [ legacy-api, ocean-api ]
+    steps:
+      - uses: actions/checkout@230611dbd0eb52da1e1f4f7bc8bb0c3a339fc8b7
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@27d0a4f181a40b142cce983c5393082c365d1480
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@94ab11c41e45d028884a99163086648e898eed25
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@6af3c118c8376c675363897acf1757f7a9be6583
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Resolve Tags
+        uses: actions/github-script@9ac08808f993958e9de277fe43a64532a609130e
+        id: tags
+        env:
+          APP: ${{ matrix.app }}
+        with:
+          script: return require('./.github/scripts/release-tags.js')({ context })
+          result-encoding: string
+      
+      - name: Build & Publish
+        uses: docker/build-push-action@7f9d37fa544684fb73bfe4835ed7214c255ce02b
+        with:
+          push: true
+          build-args: APP=${{ matrix.app }}
+          platforms: |
+            linux/amd64
```

Comment:
This should be separated into a matrix because we want to run them concurrently so that it's faster in parallel.

For speed to deployment - we should not need to wait for both platforms to be built and then pushed.

Reply:
Makes sense, will update.